src/reducers/paths.reducer.js

```diff
@@ -32,7 +37,13 @@ export default (state: State = initialState, action: Action) => {
         [project.guppy.id]: path || getDefaultPath(project.guppy.id),
       };
     }
-
+    case SET_DEFAULT_PROJECT_PATH: {
+      const { path } = action;
+      return {
+        ...state,
+        defaultProjectPath: path,
```

Comment:
The current structure for this reducer's state is a map-like object:

```js
{
  'first-react-app': 'path/to/project/',
  'second-react-app': 'path/to/project/',
}
```

Because of that, I think it's a bit weird if there's also a `defaultProjectPath` living as a sibling to these projects. For one thing, if someone oddly chooses to name their project `defaultProjectPath`, it would break.

I think we'd need to restructure the reducer so that it has this kind of shape:

```
{
  defaultHome: '~/guppy-projects',
  byId: {
    'first-react-app': 'path/to/project/',
    'second-react-app': 'path/to/project/',
  },
}
```

This way, it's clear that there's a distinction between project paths and the default value. I like the idea of changing the names so that the "parentPath" just becomes the "home" for the projects.

It does mean that some refactoring would be necessary: `getPathForProjectId` selector would need to be updated to use the new structure.